modules/cnf-image-based-upgrade.adoc

```diff
@@ -56,7 +56,10 @@ For the `Prep` stage, you specify the following upgrade details in the `ImageBas
 * extra manifests to apply and custom catalog sources to retain after the upgrade, if any
 
 Then, based on what you specify, the {lcao} prepares for the upgrade without impacting the current running version.
-During this stage, the {lcao} ensures that the target cluster is ready to proceed to the `Upgrade` stage by checking if it meets certain conditions and pulls the seed image to the target cluster with additional container images specified in the seed image.
+During this stage, the {lcao} ensures that the target cluster is ready to proceed to the `Upgrade` stage by checking if it meets certain conditions.
+The Operator pulls the seed image to the target cluster with additional container images specified in the seed image.
+The {lcao} checks if there is enough space on the container storage disk and if necessary, the Operator deletes unpinned images until the disk usage is below the specified threshold.
+For more information about how to configure or disable the cleaning up the container storage disk, see __.
```

Comment:
@amolnar-rh don't forget the "___." thing at the end of the sentence.  Too bad there's no link for me to check.

Reply:
@GroceryBoyJr Updated this now and added a link in the additional resources! PTAL :)